app/src/main/res/values/strings.xml

```diff
@@ -59,6 +59,9 @@
   <string name="settings_notifications_about_to_start_summary_on">Show notifications for favourite events starting soon</string>
   <string name="settings_notifications_about_to_start_summary_off">Do not show notifications for favourite events starting soon</string>
   <string name="settings_message_signed_out">You have been signed out.</string>
+  <string name="about_title">About</string>
+  <string name="settings_account_not_logged_in">Not logged in</string>
+  <string name="settings_header_not_signed_in">You’re not signed in</string>
```

Comment:
There is no account data displayed when there is no account. Do you mean hiding the email item maybe?